mkdocs/docs/backends/gpu-vulkan.md

```diff
@@ -1,10 +1,149 @@
-# Vulkan GPU backend
+# Vulkan GPU HAL Driver
 
-<!-- TODO(??): overview, when to use CPU vs GPU -->
+IREE can accelerate model execution on GPUs via [Vulkan][vulkan], a low-overhead
+graphics and compute API. Vulkan is cross-platform: it is available on many
+operating systems, including Android, Linux, and Windows. Vulkan is also
+cross-vendor: it is supported by most GPU vendors, including AMD, ARM, Intel,
+NVIDIA, and Qualcomm.
+
+<!-- TODO(??): when to use CPU vs GPU -->
+
+<!-- TODO(??): support matrix of different GPUs -->
 
 ## Prerequisites
 
-<!-- TODO(??): build from source, vcpkg -->
+In order to use Vulkan to drive the GPU, you need to have a functional Vulkan
+environment. IREE requires Vulkan 1.1 on Android and 1.2 elsewhere. It can be
+verified by the following steps:
+
+=== "Android"
+
+    Android mandates Vulkan 1.1 support since Android 10. You just need to
+    make sure the device's Android version is 10 or higher.
+
+=== "Linux"
+
+    Run the following command in a shell:
+
+    ``` shell
+    vulkaninfo | grep apiVersion
+    ```
+
+    If `vulkaninfo` does not exist, you will need to [install the latest Vulkan
+    SDK][vulkan-sdk]. For Ubuntu 18.04/20.04, installing via LuarG's package
+    repository is recommended, as it places Vulkan libraries and tools under
+    system paths so it's easy to discover.
```

Comment:
typo: LunarG

also not sure about keeping this whole "For Ubuntu ..." comment, we can let the official documentation give recommendations on how to install

Reply:
There is no recommendation there so this. I personally feel putting it under a directory and use `VULKAN_SDK` is causing quite some issues (as we need to set `LD_LIBARY_PATH` etc. otherwise layers cannot be properly discovered).  An suggestion here might save us some questions and configuration debugging later if somebody ask us why things don't work for them. :)